--- a/src/external/miniaudio.h	2024-07-16 08:51:08.388875603 -0400
+++ b/src/external/miniaudio.h	2024-07-16 09:11:43.818501802 -0400
@@ -28342,8 +28342,8 @@
     ma_result result;
 #ifndef MA_NO_RUNTIME_LINKING
     const char* libasoundNames[] = {
-        "libasound.so.2",
-        "libasound.so"
+        "@linuxAudioLibs@/libasound.so.2",
+        "@linuxAudioLibs@/libasound.so"
     };
     size_t i;
 
@@ -30856,8 +30856,8 @@
     ma_result result;
 #ifndef MA_NO_RUNTIME_LINKING
     const char* libpulseNames[] = {
-        "libpulse.so",
-        "libpulse.so.0"
+        "@linuxAudioLibs@/libpulse.so",
+        "@linuxAudioLibs@/libpulse.so.0"
     };
     size_t i;
 
